core/src/main/java/com/linecorp/armeria/server/http/dynamic/ResponseConverter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.dynamic;
+
+import com.linecorp.armeria.common.http.HttpResponse;
+
+/**
+ * Converts given {@code T} type object into {@link HttpResponse}.
```

Comment:
Oh my god, it is a residue I left when I was trying to make `ResponseConverter` a generic interface. After some meaningless efforts, I decided to make it non-generic. I will fix the Javadoc comments.